packages/extended-text-entry/package.json

```diff
@@ -20,7 +20,8 @@
     "lodash.throttle": "^4.1.1",
     "prop-types": "^15.6.1",
     "react": "^16.8.1",
-    "react-dom": "^16.8.1"
+    "react-dom": "^16.8.1",
+    "uuid": "^8.3.2"
```

Comment:
My understanding is that you added this to generate uniq ids for the selections. Do you think we need to have a library for this, ok could we use a concatenation of itemId + new Date().getTime()?

Reply:
Not really. I removed the library and created a custom id based on start and end position of a annotation and new Date().getTime()